modern/gcc/Dockerfile

```
@@ -65,5 +66,5 @@ RUN sudo rm -rf /usr/lib/gcc/x86_64-linux-gnu/* \
     && sudo update-alternatives --install /usr/local/bin/c++ c++ /usr/local/bin/g++ 100 \
     && sudo rm /etc/ld.so.cache \
     && sudo ldconfig -C /etc/ld.so.cache \
-    && conan profile new --detect --force default \
-    && conan profile update settings.compiler.libcxx=libstdc++11 default
+    && sudo chmod +x /opt/conan/gcc_bootstrap.sh \
+    && /opt/conan/gcc_bootstrap.sh
```

We can remove this file from the final image

```suggestion
    && /opt/conan/gcc_bootstrap.sh \
    && rm /opt/conan/gcc_bootstrap.sh
```

---

done